packages/stream-node/reader.js

```diff
@@ -37,12 +37,10 @@ export const makeNodeReader = input => {
       return iterator.next();
     },
     async return() {
-      input.destroy();
       assert(iterator.return);
       return iterator.return();
     },
-    async throw(error) {
-      input.destroy(error);
+    async throw() {
```

Comment:
If we're to keep the wrapper here, I actually like the explicit `destroy(error)` which passes the error to the stream.

Reply:
Posted a follow-up commit.